tools/crawler/.sonarrc

```diff
@@ -0,0 +1,31 @@
+{
+    "connector": {
+        "name": "jsdom",
+        "options": {
+            "waitFor": 1000
+        }
+    },
+    "formatters": "summary",
+    "rulesTimeout": 120000,
+    "rules": {
+        "apple-touch-icons": "warning",
+        "axe": "off",
```

Comment:
"off" ?

Reply:
Yes, `axe` timesout a lot with `jsdom` and it slows down the process a lot. The idea of the crawler is mainly te fetch websites, resources, and do the traversing to find reliability problems so this should be ok.